src/jail_conf.c

```diff
@@ -85,9 +85,14 @@ jail_conf_parse(
 
     str = g_key_file_get_string(f, SECTION, KEY_EXEC, NULL);
     if (str) {
-        g_free(priv->exec);
-        conf->exec = priv->exec = str;
-        GDEBUG("  %s=%s", KEY_EXEC, conf->exec);
+        if (str[0] != '/') {
+            /* Protecting user from bad decisions */
+            GWARN("[sailjail] " KEY_EXEC "value '%s' must be an absolute path, ignoring!", str);
+        } else {
+            g_free(priv->exec);
+            conf->exec = priv->exec = str;
```

Comment:
Nothing wrong with the change here, but I had missed this "use something else instead of firejail" config option altogether.

Given that everything sailjail does is geared towards executing something that is exactly like firejail - what possible use this could have?

Reply:
I don't know and I think I'm not the person to ask that. You should discuss monich about it.

The only use I can think of would be to test alternative builds of firejail but even that seems quite unusual.